test/ownership/ownership_test.dart

```diff
@@ -5,6 +5,7 @@
 
 import 'dart:typed_data';
 
+import 'package:sqlite3/common.dart';
 import 'package:sqlite3/sqlite3.dart';
```

Comment:
Do we need both imports here?

Reply:
@ricardobrg Something that your review points out is that in order downstream, libraries will likely have to import both '`sqlite3/common.dart` and `sqlite3/sqlite3.dart`. The former provides us `CommonDatabase`, and the later provides use the `sqlite` property.